select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java

```diff
@@ -53,7 +55,6 @@
 	private String initSelection;  //TODO this will not work
 	private String query;
 	private String width;
-	private String theme;
```

Comment:
I'm afraid this change breaks original meaning of 'theme' described here: https://select2.github.io/examples.html#themes-templating-responsive-design